tests/frontend_quality_yellow_lab_tools.py

```diff
@@ -28,12 +32,12 @@ def run_test(langCode, url, device='phone'):
 
 	print(_("TEXT_RUNNING_TEST"))
 
-	r = requests.post('https://yellowlab.tools/api/runs', data = {'url':url, "waitForResponse":'true', 'device': device})
+	r = requests.post('{}/api/runs'.format(ylt_server_address), data = {'url':url, "waitForResponse":'true', 'device': device})
```

Comment:
```suggestion
	r = requests.post('{0}/api/runs'.format(ylt_server_address), data = {'url':url, "waitForResponse":'true', 'device': device})
```

Reply:
@marcusosterberg  Can we change it to the suggested version? If yes, I will approve it with that change :)